docs/guides/migrate-plugin.md

```diff
@@ -136,6 +114,4 @@ Alternatively, use the `nextflow plugin create` command to re-create your plugin
 
 The Nextflow Gradle plugin supports publishing plugins from the command line. See {ref}`gradle-plugin-publish` for more information.
 
-:::{warning}
-Once you migrate to the Gradle plugin, you will no longer be able to publish to the legacy plugin index. See the [Migration timeline](#migration-timeline) for more information.
-:::
+The legacy plugin index is **closed to new pull requests**. It will be kept up-to-date with the plugin registry and will remain available indefinitely, in order to ensure continued support for older versions of Nextflow.
```

Comment:
```suggestion
The legacy plugin index is **closed to new pull requests**. It will be kept up-to-date with the plugin registry and will remain available indefinitely to ensure continued support for older versions of Nextflow.
```